### ISSUE TITLE
shader HLSL 5.0 or higher error

### ISSUE THREAD

**Issue by 3coms** [TheGameCreators/GameGuruRepo#686]
When I load old maps, some models use old shaders, this generates an error about "HLSL 5.0 or higher".

I understand why this happens and what I have to do.

The problem is that GG immediately crash up and I have to close the program from the task manager.

This makes it difficult to identify which models are affected, in order to solve the problem.
Scanning the folder containing the models one by one, does not solve the problem, since we usually also use models from different folders.

I wonder if it would be possible in such cases to load a default shader such as "entity_basic.fx"  and skip the error.
In this way the map is loaded normally, GG works as normal, and the user can more easily identify the objects that use old shaders, and change this shader for a more updated and compatible with GG.

**Comment by 3coms** [comment]
![hlsl_5](https://user-images.githubusercontent.com/47690174/74103599-9e61dc80-4b4d-11ea-902f-7246f3e9cffb.jpg)

**Comment by 3coms** [comment]
Still remain in last build (operation bug free).
t looks like GG reading old version of fpm rather than the new one wirh shaders fixed. You has to open new map in save it, in order to change the files in testmap, or GG remain reading the old fpm version with the offended items and does not matter if you has to fixed it.

I've revise all my entitybank and fix old the fpe files point to wrong shader, so why GG still talking about "waterfall.fx" it does not exist in any fpe in my entitybank, I've wrote a script than scan all my fpe files and search for wrong shaders and replace it by default one.

Finally I've edited ele file and fixed all the fx offended lines and now the map works like a charm.

I want to make clear than I don't want to use these deprecated shaders, just recovery  larger maps using those shaders.

Goal here is to avoid GG crashing when error raise, maybe uploading default fx instead.

![waterfall3FX test](https://user-images.githubusercontent.com/47690174/78087915-d4d80d00-73c2-11ea-8500-b3c1013b0c0d.png)
